br/pkg/stream/rewrite_meta_rawkv.go

```diff
@@ -104,6 +105,7 @@ func NewDBReplace(name string, newID DownstreamID) *DBReplace {
 // NewSchemasReplace creates a SchemasReplace struct.
 func NewSchemasReplace(
 	dbMap map[UpstreamID]*DBReplace,
+	idMapNeedConstruct bool,
```

Comment:
minor: idMapNeedConstruct can be renamed as needConstructIdMap

Reply:
done